README.md

```diff
@@ -21,12 +21,27 @@ I like emoji. I like ’em a lot. Programming, code, geeks/nerds, open source, a
 1. **IMPERATIVE** ↓
 	- Make your Git commit messages imperative.
 	- Write commit message like you're giving an order.
-	- E.g. Use ✅ `Add` instead of ❌ `Added`.
-	- E.g. Use ✅ `Create` instead of ❌ `Creating`.
+	
+	_For Example:_
+	
+	  * Use `✅ Add` instead of `❌ Added`.
+	  * Use `✅ Create` instead of `❌ Creating`.
+	
 1. **RULES** ↓
 	- Small number of categories — easy to memorize.
 	- Nothing more nothing less.
-	- E.g. `📦 NEW`,`👌 IMPROVE`,`🐛 FIX`,`📖 DOC`,`🚀 RELEASE`, and `✅ TEST`
+	
+    _For Example:_
+	
```

Comment:
Keeping them in one line makes the rules look small and easy to digest. So, no to this format.